monitoring/grafana/bb-scheduler.jsonnet

```diff
@@ -9,17 +9,18 @@ simpledash.dashboard(
       label='Instance name',
       selectionStyle=simpledash.selectMultiple,
     ),
-    // TODO: Make it possible to filter the dashboard by platform. This
-    // doesn't work right now, as Grafana doesn't escape double quotes
-    // in regex patterns properly.
-    /*
+    // Platform properties contains double quotes, so use regex patterns
```

Comment:
nit: "use" here is not needed, I would prefer "so the regex patterns need to be signel-quoted ..."

Reply:
Done in the previous commit.